### ISSUE TITLE
Behaviour after homey restart

### ISSUE THREAD

**Issue by PESK01** [timkouters/nl.timkouters.domitech#6]
Hi,
After every restart/reboot of Homey, my Domitec bulbs starts. They look "off" in Homey app, but to really stop them, i need to start/stop every light once to make them "off". This behaviour started after upgrade to Homey 2. None of my other Z-wave does this.
Best regards
//Peter Skånberg

**Comment by timkouters** [comment]
Hi Peter,

Thanks for letting me know.
I have somewhat the same issue at home since about a week.
My issue is that even though I turn off the lamps in the evening at 23:00, in the morning they are turned on again but in the app they are off.
I'll try to investigate somewhere next week what is exactly happening; since I do not have time this weekend to do so.

I have updated my homey to 2.0 already for 4 weeks and back then I did not have this issue.
So my suspicion is that Athom has added some new behavior in the last week.

Kind regards,
Tim

**Comment by froh2018** [comment]
Hi Tim,
Turning on the bulbs after a restart of Homey 2.0 seems to be a side effect of your latest update (remember dim level). I've got four bulbs and after I experienced the same behaviour as Peter I have tested various settings. If remember dim level is turned on the bulb to will turn on after an update of Homey.  If remember dim level is turned off the bulb will not turn on after an update of Homey.
Kind regards,
Kees
